Update README_pt-BR.md

Space-Cowb0y opened the pull request:
Adicionando novas funcionalidades descritas no README.md em inglês e as traduzindo.
Corrigindo exemplos da tabela que estavam um pouco confusos.

Adding new translations.
Fixing table examples that were a bit confusing.

Space-Cowb0y committed:
Update README_pt-BR.md

Adicionando novas funcionalidades descritas no README.md em inglês e as traduzindo.
Corrigindo exemplos da tabela que estavam um pouco confusos.

dittrichlucas commented on README_pt-BR.md:
```diff
@@ -64,25 +64,29 @@ Toda a sintaxe pode ser encontrada [aqui](https://daringfireball.net/projects/ma
-| Listas Ordenadas  	| 1. Item a    2. Item b 	| <ol><li>itema</li><li>itemb</li><li>itemc</li><li>itemd</li></ol>  	|
-| Listas mistas 	| 1. Item 1      * item 1a 	|  <ol><li>itema</li></ol><ul><li> item1</li></ul>	|
+| Listas não ordenadas 	| \* Item1     \*Item 2 	| <ul><li>item1</li><li>item2</li> 	|
+| Listas Ordenadas  	| 1. Item a    2. Item b 	| <ol><li>itema</li><li>itemb</li>  	|
```
I think it's better to keep a pattern. Since the previous items start with a lowercase letter, what do you think about keeping this here?

dittrichlucas commented on README_pt-BR.md:
```diff
@@ -64,25 +64,29 @@ Toda a sintaxe pode ser encontrada [aqui](https://daringfireball.net/projects/ma
    <br></br>
-   O mesmo vale para **Bloco de codigo/Destaque de sintaxe**. É um pouco parecido com essa imagem :
-   ![](http://i.imgur.com/z8KrxAz.png).</p>
+   O mesmo vale para **Bloco de código/destaque de sintaxe**. Sua aprarencia é igual a da imagem abaixo:
```
aparência*

Space-Cowb0y committed:
solve misspelling

solve misspelling in line 84

Space-Cowb0y committed:
Correct line 145

changing mermaid code in  line 145 "utl" to "url"

Space-Cowb0y committed:
Keeping the fist column in the table in uppercase

to follow a pattern like said in https://github.com/luong-komorebi/Markdown-Tutorial/pull/16#discussion_r949458253

Space-Cowb0y committed:
Update with note and warning

to follow the commit on https://github.com/luong-komorebi/Markdown-Tutorial/pull/15#issue-1331030898

Space-Cowb0y committed:
misspelling correction

Space-Cowb0y committed:
changing "marcação" to "sintaxe"

dittrichlucas commented:
LGTM 👍🏼
